modules/nf-core/vclust/cluster/meta.yml

```
@@ -0,0 +1,103 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "vclust_cluster"
+description: Vclust cluster performs threshold-based clustering by assigning a
+  genome sequence to a cluster if its similarity (e.g., ANI) to the cluster
+  meets or exceeds a user-defined threshold.
+keywords:
+  - cluster
+  - virus
+  - filter
+  - contig
+  - scaffold
+  - phage
+tools:
+  - "vclust":
+      description: "Fast and accurate tool for calculating ANI and clustering virus
+        genomes and metagenomes."
+      homepage: "https://afproject.org/vclust/"
+      documentation: "https://github.com/refresh-bio/vclust/wiki"
+      tool_dev_url: "https://github.com/refresh-bio/vclust"
+      doi: "10.1038/s41592-025-02701-7"
+      licence: ["GPL v3-or-later"]
+      identifier: "biotools:vclust"
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1' ]`
+    - tsv:
+        type: file
+        description: Sorted TSV file, generated by `vclust align`
+        pattern: "*.tsv"
+        ontologies:
+          - edam: "http://edamontology.org/format_3475" # TSV
+  - - meta2:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1' ]`
+    - ids:
+        type: file
+        description: File containing sequence IDs and their number of parts,
+          generated by `vclust align`
+        pattern: "*.ids.tsv"
+        ontologies:
+          - edam: "http://edamontology.org/format_3475" # TSV
+  - - metric:
+        type: string
+        description: |
+          Similarity metric for clustering choices: 'tani','gani' or 'ani'
```

```suggestion
          Similarity metric for clustering choices: 'tani','gani' or 'ani'
        pattern: "tani|gani|ani"
```

OR however lists should be formatted